tests/server/server-executeCommand-002.phpt

```diff
@@ -0,0 +1,78 @@
+--TEST--
+MongoDB\Driver\Server::executeCommand() takes a read preference
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; NEEDS("REPLICASET"); ?>
```

Comment:
Is this just a matter of supporting certain IDEs and editor plugins? I personally never had such functionality, so I didn't notice this.